src/webview/webkitgtk/mod.rs

```diff
@@ -343,22 +335,19 @@ impl InnerWebView {
     };
 
     // Initialize message handler
-    let mut init = String::with_capacity(115 + 20 + 22);
-    init.push_str("Object.defineProperty(window, 'ipc', {value: Object.freeze({postMessage:function(x){window.webkit.messageHandlers[\"");
-    init.push_str(&window_hash);
-    init.push_str("\"].postMessage(x)}})})");
-    w.init(&init)?;
+    w.init("Object.defineProperty(window, 'ipc', { value: Object.freeze({ postMessage: function(x) { window.webkit.messageHandlers['ipc'].postMessage(x) } }) })")?;
```

Comment:
Fixup: Changed this to match what is in the dev branch currently.

Reply:
this is a bit of breaking change, let's revert
```suggestion
    let mut init = String::with_capacity(115 + 20 + 22);
    init.push_str("Object.defineProperty(window, 'ipc', {value: Object.freeze({postMessage:function(x){window.webkit.messageHandlers[\"");
    init.push_str(&window_hash);
    init.push_str("\"].postMessage(x)}})})");
    w.init(&init)?;
```